src/fish/install.sh

```diff
@@ -118,6 +119,24 @@ if [ "${FISHER}" = "true" ]; then
   fish -c "fisher -v"
 fi
 
+# Install Fisher plugins
+if [ -n "${FISHER_PLUGINS}" ]; then
```

Comment:
fisher should be installed.

```suggestion
if [ "${FISHER}" = "true" ] && [ -n "${FISHERPLUGINS}" ]; then
```